devnet/alice.Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM golang:1.17
+
+ARG DD_API_KEY=somekey
+
+ENV GO111MODULE=on
+ENV DD_API_KEY=${DD_API_KEY}
+
+RUN echo $DD_API_KEY
+
+RUN ["sh", "-c", "DD_AGENT_MAJOR_VERSION=7 DD_INSTALL_ONLY=true DD_API_KEY=${DD_API_KEY} DD_SITE=\"datadoghq.com\" bash -c \"$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)\""]
+
+WORKDIR /gossamer
+
+COPY . . 
+
+RUN go run devnet/cmd/update-dd-agent-confd/main.go -n="gossamer.local.devnet" -t="key:alice" > /tmp/conf.yaml
+RUN mv /tmp/conf.yaml /etc/datadog-agent/conf.d/openmetrics.d/conf.yaml
+RUN service datadog-agent start
+
+RUN go get ./...
```

Comment:
You should cache Go dependencies with something similar to

https://github.com/ChainSafe/gossamer/blob/development/Dockerfile#L27-L35